diff --git a/setup.py b/setup.py
index 249051f..272dca6 100644
--- a/setup.py
+++ b/setup.py
@@ -19,10 +19,10 @@ install_requires = [
     # https://github.com/shazow/urllib3/issues/556. This requirement can be
     # relaxed to 'requests[security]>=2.4.1', however, less useful errors
     # will be raised for some network/SSL errors.
-    'requests[security]>=2.10',
+    'requests',
     # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
     # will tolerate; see #2599:
-    'setuptools>=1.0',
+    'setuptools',
     'six',
 ]
 
