diff --git a/acme/client_test.py b/acme/client_test.py
index 33e80aa..c972004 100644
--- a/acme/client_test.py
+++ b/acme/client_test.py
@@ -568,6 +568,7 @@ class ClientNetworkTest(unittest.TestCase):
             'GET', 'http://example.com/',
             verify=mock.ANY, headers={'User-Agent': 'foo2'})
 
+    @unittest.skip("Fedora python-mock missing asset_called_once bz#1244145")
     def test_del(self):
         sess = mock.MagicMock()
         self.net.session = sess
